shared/src/store/action-reducers/vehicle.ts

```diff
@@ -100,8 +105,8 @@ export class LoadVehicleAction implements Action {
 
     @IsString()
     public readonly elementToBeLoadedType!:
-        | 'material'
-        | 'patient'
+        | 'materials'
+        | 'patients'
```

Comment:
Yes! Good catch.

But I'm a bit confused that I didn't encounter any error when trying to migrate any of the states...

Reply:
There is no error because the validation is broken, and it is translated to no behavior, so nothing gets loaded (apart from `personnel` which didn't change).

Reply:
> There is no error because the validation is broken, and it is translated to no behavior, so nothing gets loaded (apart from `personnel` which didn't change).

I wouldn't say the validation is broken - it just isn't implemented yet ^^
But your explanation makes complete sense.